CHANGES/11114.misc.rst

```diff
@@ -0,0 +1 @@
+Downgraded the logging level for connector close errors from ERROR to DEBUG, as these are expected behavior with TLS 1.3 connections -- by :user:`bdraco`.
```

Comment:
Wait, is it not expected that 1.3 connections close correctly? I thought it was  badly implemented servers that don't close the connection?

Reply:
Agreed here. This seems like a bad idea. It's basically suppressing a log entry that is alerting you to 100ms additional latency added by the server.